notebooks/232-clip-language-saliency-map/232-clip-language-saliency-map.ipynb

```diff
@@ -73,6 +73,16 @@
   {
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.        infer_request: InferRequest,  # inferente result
</pre><p>this output does not correspond to this cell, could you please preserve opportunity to launch without gradio</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1142/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='34'/>

Reply:
Done